src/vs/workbench/services/workingCopy/browser/workingCopyBackupTracker.ts

```diff
@@ -32,27 +32,27 @@ export class BrowserWorkingCopyBackupTracker extends WorkingCopyBackupTracker im
 	protected onFinalBeforeShutdown(reason: ShutdownReason): boolean {
 
 		// Web: we cannot perform long running in the shutdown phase
-		// As such we need to check sync if there are any dirty working
+		// As such we need to check sync if there are any modified working
 		// copies that have not been backed up yet and then prevent the
 		// shutdown if that is the case.
 
-		const dirtyWorkingCopies = this.workingCopyService.dirtyWorkingCopies;
-		if (!dirtyWorkingCopies.length) {
-			return false; // no dirty: no veto
+		const modifiedWorkingCopies = this.workingCopyService.modifiedWorkingCopies;
+		if (!modifiedWorkingCopies.length) {
+			return false; // no unsaved changes: no veto
```

Comment:
The comments refer to "unsaved changes" but I think for better understanding can we refer to this as "modified"?